### PR TITLE
Follow up Pull Request Revert issues are no longer created

### PR DESCRIPTION
**Pull request by ricardoamador**
This pull request removes the feature of creating a review issue when a revert pull request is generated. Previously a tracking issue was created so that a flutter-hacker member would follow up on a review of the reverted request. This has not been a popular feature and has caused more of a headache than anything else.

*List which issues are fixed by this PR. You must list at least one issue.*
Feedback the devs.

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: cron.yaml
Comment (yusuf-goog) [inline cron.yaml:57]: newline at EOF

**Review by yusuf-goog — APPROVED** [review]

**Comment by godofredoc** [comment]
@Hixie FYI you're going to like this PR

File: cron.yaml
Comment (ricardoamador) [reply cron.yaml]: It is odd, I have the newline in my editor and when it gets sent here it does not show. Maybe github is not showing the newline on EOF?

**Comment by Hixie** [comment]
ah cool. Does it just require a normal review before landing now, like other patches?

**Comment by ricardoamador** [comment]
No you would still not need a review up front to merge it. Otherwise it would just be a regular commit from the point of view of autosubmit. 
 I am in the process of reworking this entirely where you would only need to add the label to a closed and merged pull request; then the bot would take care of everything.

**Comment by Hixie** [comment]
do we verify that the commit is precisely a revert of a (very recent) PR?
can we make this ping a channel on discord?
we should never have any code (revert or not) land without two party approval.

**Comment by ricardoamador** [comment]
We verify the change counts in the two pull requests.
Yes we can make this ping discord. I have run a quick test from my local to
ping the tree status channel and it works fine. Just not sure about when it
is deployed to gcp but I can test that.
If that is the case then there is no special path needed for revert.
Everything would just go through autosubmit.

**Comment by ricardoamador** [comment]
To add to this, there has been pushback from the developers that say they
need to be able to revert changes faster which is why the tracking review
issue on reverts was created. So there would be visibility on the revert
and to perform a review. We can still allow that but enforce a two party
review.

**Comment by Hixie** [comment]
the change counts alone isn't going to verify it's a perfect revert. it would be easy to sneak in a vulnerability if all that has to match is the change counts.

**Comment by ricardoamador** [comment]
Yeah, I agree. That is why I am getting to an update to prevent the user
from even needing to do anything aside from adding a label.

**Comment by Hixie** [comment]
to be clearer:

- if we can 100% guarantee that a revert is a precise revert of a PR that landed in the last ~24 hours then I think it's ok to let that go through with just one party approval (the flutter-hackers member adding the label). In theory people could carefully construct a pair of PRs where reverting the first after landing the second opens up a vulnerability but frankly code review isn't going to catch that anyway.
- anything else imho should just use the normal two-party review system we have for any PR. If it's older than a day then it's clearly not that urgent any more and can wait the extra 15 minutes to find a reviewer. If it's more complicated than a straight review then similarly it can get a more thorough review.

If we can make the revert system just be "add a label to a recent PR, and that PR will get autoreverted by a bot whose integrity we can verify" then that sounds fantastic. It would guarantee the revert is correct. (I would recommend still posting an FYI to the #tree-status channel, since people might want to know what is happening to fix the tree.) (Also we should still limit that to "recent" PRs, for some definition of "recent" that's something like "past 24 hours".)

**Pull request merged**
